l10n_es_aeat_mod303_special_prorate/data/tax_code_map_mod303_data.xml

```diff
@@ -0,0 +1,340 @@
+                (4, ref('l10n_es_special_prorate.account_tax_template_p_priva4_ic_bc')),
+                (4, ref('l10n_es_special_prorate.account_tax_template_p_priva4_sp_in')),
+                (4, ref('l10n_es_special_prorate.account_tax_template_p_priva4_ic_bi')),
+                (4, ref('l10n_es_special_prorate.account_tax_template_p_priva10_ic_bc')),
+                (4, ref('l10n_es_special_prorate.account_tax_template_p_priva10_sp_in')),
+                (4, ref('l10n_es_special_prorate.account_tax_template_p_priva10_ic_bi')),
+                (4, ref('l10n_es_special_prorate.account_tax_template_p_priva21_ic_bc')),
+                (4, ref('l10n_es_special_prorate.account_tax_template_p_priva21_sp_in')),
+                (4, ref('l10n_es_special_prorate.account_tax_template_p_priva21_ic_bi')),
+            ]"
+        />
+
+    </record>
+
+    <record
+        id="l10n_es_aeat_mod303.aeat_mod303_202107_map_line_14_purchase"
+        model="l10n.es.aeat.map.tax.line"
+    >
+        <field
+            name="tax_ids"
+            eval="[
+                (4, ref('l10n_es_special_prorate.account_tax_template_p_priva4_ic_bc')),
+                (4, ref('l10n_es_special_prorate.account_tax_template_p_priva4_sp_in')),
+                (4, ref('l10n_es_special_prorate.account_tax_template_p_priva4_ic_bi')),
+                (4, ref('l10n_es_special_prorate.account_tax_template_p_priva10_ic_bc')),
+                (4, ref('l10n_es_special_prorate.account_tax_template_p_priva10_sp_in')),
+                (4, ref('l10n_es_special_prorate.account_tax_template_p_priva10_ic_bi')),
+                (4, ref('l10n_es_special_prorate.account_tax_template_p_priva21_ic_bc')),
+                (4, ref('l10n_es_special_prorate.account_tax_template_p_priva21_sp_in')),
+                (4, ref('l10n_es_special_prorate.account_tax_template_p_priva21_ic_bi')),
+           ]"
+        />
+    </record>
+
+    <record
+        id="l10n_es_aeat_mod303.aeat_mod303_202107_map_line_15_purchase"
+        model="l10n.es.aeat.map.tax.line"
+    >
+        <field
+            name="tax_ids"
+            eval="[
+                (4, ref('l10n_es_special_prorate.account_tax_template_p_priva4_ic_bc')),
+                (4, ref('l10n_es_special_prorate.account_tax_template_p_priva4_sp_in')),
+                (4, ref('l10n_es_special_prorate.account_tax_template_p_priva4_ic_bi')),
+                (4, ref('l10n_es_special_prorate.account_tax_template_p_priva10_ic_bc')),
+                (4, ref('l10n_es_special_prorate.account_tax_template_p_priva10_sp_in')),
+                (4, ref('l10n_es_special_prorate.account_tax_template_p_priva10_ic_bi')),
+                (4, ref('l10n_es_special_prorate.account_tax_template_p_priva21_ic_bc')),
+                (4, ref('l10n_es_special_prorate.account_tax_template_p_priva21_sp_in')),
+                (4, ref('l10n_es_special_prorate.account_tax_template_p_priva21_ic_bi')),
+            ]"
+        />
+    </record>
```

Comment:
```suggestion
```
This ones are wrong. I checked the v11, comparing with v14, and the map lines 10, 11, 14 and 15 shouldn't have any of these prorrates

The others are ok.

Reply:
Hummm, they do exist on v11 (the negative part of them)

![image](https://user-images.githubusercontent.com/10927087/161022568-7ff8e14c-e07e-4866-8faa-8d5ca46bb855.png)

Why do you think they're wrong?

Reply:
Sorry for the noise, I checked it wrong.

The two checks I did was:
1- The prorate taxes are linked to l10n_es taxes in v11 by using the same description field.
2- The l10n_es taxes are child taxes of some prorate taxes. (this second condition is where I did wrong query)

After correcting the query, I realize map lines 7, 9, 14, 15 and 80 lines need "IGF Repercutido" taxes.

Reply:
No problem :-)
The IGF is a manually created tax so it should be added manually as well, maybe it was lost during the migration.

Reply:
Yeah, hide I mean, but it's not hidden yet :S